templates/careers/index.html

```diff
@@ -18,80 +20,116 @@
   <img src='https://pubads.g.doubleclick.net/activity;xsp=5024608;ord=1?' width=1 height=1 border=0>
 </noscript>
 
-<section class="p-strip--image is-dark is-deep" style="background-image: url(https://assets.ubuntu.com/v1/aebc1742-01_suru2_dark_2K.png); background-position: 100% 55%; background-size: 180%; background-repeat: repeat-x;">
+<section class="p-section">
+  <div class="u-fixed-width">
+    <nav class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
```

Comment:
I'm not sure if the nav element is the best choice as there aren't any links in the list, from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav): 

"The `<nav>` HTML element represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents."

Further down:
"The semantics of the nav element is that of providing links."

Reply:
Changed to a `div`. Design doesn't want us to link to the same page, but in theory this is a list of navigation links.